### PR TITLE
Add link to SaorsaLabs template canister repo

### PR DESCRIPTION
**Pull request by NathanScotland**
Readme has been updated with a link to a Rust Template Canister repo created by Saorsa Labs. Full stop added to end of Azle link text.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @NathanScotland,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/452) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

**Comment by domwoe** [comment]
Hi @NathanScotland,

thank you for the PR, but you need accept the CLA before we can merge.

**Comment by NathanScotland** [comment]
Hi @domwoe, apologies I forgot to do the CLA bit. Hopefully it's now sorted :)

**Review by domwoe — APPROVED** [review]

**Pull request merged**
